Source/OxyPlot.Pdf/PdfReportWriter.cs

```diff
@@ -139,6 +139,7 @@ public virtual void Close()
             r.RenderDocument();
             r.PdfDocument.Save(this.Output);
             this.CleanTemporaryPlotFiles();
+            this.Output.Close();
```

Comment:
Could there be an issue if someone use the ctor with stream and already close the stream themselves? I think it would be better to remove the constructor with path, and don't take the responsibility for closing the stream here...